docs/how-to-guides/integrating-autoware/creating-maps/creating-vector-map/.pages

```diff
@@ -0,0 +1,8 @@
+nav:
+  - index.md
+  - Lanelet 2: lanelet2
```

Comment:
Looking at the [official repository's documentation](https://github.com/fzi-forschungszentrum-informatik/Lanelet2), it seems we don't need space between Lanelet and 2.
```suggestion
  - Lanelet2: lanelet2
```

Reply:
Done :+1: